release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -111,17 +111,45 @@ With this release, you can prevent the `kubeadmin` password from being displayed
 [id="ocp-4-16-web-console_{context}"]
 === Web console
 
+[id="ocp-4-16-web-console-language"]
+==== Language support for French and Spansih
+With this release, French and Spanish are suported in the web console. You can update the language in the webconsole from the *Language* list on the *User Preferences* page.
+
+[id="ocp-4-16-patternfly4-deprecated"]
+==== Patternfly 4 is now deprecated with 4.16
```

Comment:
```suggestion
==== Patternfly 4 is now deprecated
```

Reply:
Keeping the version but changing to {product-version} want to avoid  any confusion about what versions are available where.